Assets/MixedRealityToolkit/Inspectors/ClippingPrimitiveInspector.cs

```diff
@@ -18,10 +18,7 @@ public abstract class ClippingPrimitiveEditor : UnityEditor.Editor
         /// Notifies the editor that this object has custom frame bounds.
         /// </summary>
         /// <returns>True for all clipping primitives.</returns>
```

Comment:
return Comment probably needs to be updated? why override in sub classes?

Reply:
Good catch! Yeah, I wish I could just keep one implementation in the base but Unity was not respecting it (it wasn't focusing correctly when pressing 'F'). So, begrudgingly I moved it to the derived classes.